src/Kis/Kis.hs

```diff
@@ -22,23 +23,46 @@ import Kis.Model
 import Kis.Time
 
 data KisAction a where
-    CreateBed :: String -> KisAction BedId
-    CreatePatient :: Patient -> KisAction PatientId
+    CreateBed :: String -> KisAction (Maybe BedId)
+    CreatePatient :: Patient -> KisAction (Maybe PatientId)
```

Comment:
Das sieht für mich nicht sinnvoll aus. Der Maybe-Typ in den anderen Aktionen resultiert daher, dass die Datenbank-Transaktionen fehlschlagen dürfen, weil ein Patient zu einer Id nicht existiert etc. CreateBed sollte im Regelfall nicht fehlschlagen.